Decred Wallet/Features/More/More Features/Security Tools/Sign Message/SignMessageViewController.swift

```diff
@@ -0,0 +1,256 @@
+//
+//  SignMessageViewController.swift
+//  Decred Wallet
+//
+//  Created by Suleiman Abubakar on 30/12/2019.
```

Comment:
copyright:
// Copyright (c)2019 The Decred developers
// Use of this source code is governed by an ISC
// license that can be found in the LICENSE file.

Reply:
2020